.scripts/build_steps.sh

```diff
@@ -33,7 +34,7 @@ conda build "${RECIPE_ROOT}" -m "${CI_SUPPORT}/${CONFIG}.yaml" \
     --clobber-file "${CI_SUPPORT}/clobber_${CONFIG}.yaml"
 
 if [[ "${UPLOAD_PACKAGES}" != "False" ]]; then
-    upload_package "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
+    upload_package  "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
```

Comment:
Why a space here?

Reply:
This is jinja2. It removes some extra code for the output validation stuff and appears to have left an extra space. Likely can be fixed by the right combination of minus signs and percent signs. Feel free to make a PR!